application/config/config-defaults.php

```diff
@@ -78,6 +78,7 @@
 $config['maxdumpdbrecords']          = 500; // The maximum number of records that would be ouput in one go during a database backup. Reduce this number if you're getting errors while backing up the entire database.
 $config['maxdbsizeforbackup']        = 0; // The maximum database size in MB that is backed up up by ComfortUpdate - 0 for no limit
 $config['allowexportalldb']          = 0; // Default 0 will only export prefixed tables when doing a database dump. If set to 1 ALL tables in the database will be exported (use carefully)
+$config['maxDatabaseSizeForDump']    = 256; // Maximum database size in megabytes to be able to download without errors
```

Comment:
Why this value ? It can be totally different according to server. Here it's limesurvey hosting value, right ?

My opinion : no restriction by default, LimeSurvey hosting config can set it at 256